packages/contracts/contracts/optimistic-ethereum/ovm/SafetyChecker.sol

```diff
@@ -1,12 +1,11 @@
 pragma solidity ^0.5.0;
 pragma experimental ABIEncoderV2;
 
-/* Contract Imports */
-import { ExecutionManager } from "./ExecutionManager.sol";
-
 /* Library Imports */
 import { ContractResolver } from "../utils/resolvers/ContractResolver.sol";
 
+import { console } from "@nomiclabs/buidler/console.sol";
```

Comment:
It looks like this `console` import could be removed along with the commented out code that calls it below. Do you know if the compiler is smart enough to not include imports that aren't called in the bytecode?

Reply:
Ah true! I can definitely remove this